### PR TITLE
upgrade rails gem vulnerabilities

### PR DESCRIPTION
**Pull request by isaacmg410**
Change rails version to 5.2.2.1 to fix vulnerabilities

### CODE REVIEW

**Review by tramuntanal — APPROVED** [review]
Thanks!

File: decidim-members.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'decidim-core', '~> 0.15', '>= 0.15.0'
   s.add_dependency 'pg_search', '~> 2.1.4'
-  s.add_dependency 'rails', '~> 5.2'
+  s.add_dependency 'rails', '~> 5.2.2.1'
```
Comment (tramuntanal) [inline decidim-members.gemspec:27]: Add a comment explaining why we add this and when it can be removed please.

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: decidim-members.gemspec (hunk 1 above)
Comment (isaacmg410) [reply decidim-members.gemspec:27]: Done

**Review by tramuntanal — APPROVED** [review]

**Comment by tramuntanal** [comment]
Great job Isaac!

**Pull request merged**